GettingStarted.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: default
+title: "Getting Started"
+permalink: /GettingStarted.md/
```

Comment:
do we need the .md and stuff in the url?

Reply:
Made these changes.